### DISCUSSION TITLE
How to start auto import using Nordigen (fireflyIII noobie)

### DISCUSSION THREAD

**Discussion by WoltervanDonk** [firefly-iii/firefly-iii#6391] [Firefly III] [answered]
I have nordigen setup and also a config file that works wich i have stored in the docker in 
/var/www/html/storage/configurations The file is called default conf.json. This also shows up if i do it manually
![image](https://user-images.githubusercontent.com/23165739/187534531-9e2a5f1a-9188-4c71-8f6e-18b9bc2940fb.png)

my firefly app itself runs on port 8080
my fidi importer runs on port 8081

here is the script i run to try to automatically import

sudo docker run fireflyiii/data-importer:latest \
--rm \
-e FIREFLY_III_ACCESS_TOKEN= (my token) \
-e FIREFLY_III_URL= 192.168.2.240:8080 \
-e NORDIGEN_ID= (my nordigen id) \
-e NORDIGEN_KEY= (my nordigen key) \
fireflyiii/data-importer:latest

When running this commando i get the following:
![image](https://user-images.githubusercontent.com/23165739/187533558-eabdafa7-2c56-44d4-8ca0-3e40198718f3.png)

and then it's just stuck and doesn't do anything. let it run for roughly 20 minutes. 

Does anyone know how to properly setup the auto import of Nordigen data. I run everything in docker on my Synology Nas.

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/data-importer/usage/command_line/

At the bottom.

**Reply by WoltervanDonk** [reply]
That's what i tried running but i get the same result. I run that exact command but adding my NORDIGEN_ID and NORDIGEN_KEY as in my sample.

still stuck on the same message:
![image](https://user-images.githubusercontent.com/23165739/188208906-7e20d566-50ba-416f-bb7c-67973cc09673.png)

**Reply by JC5** [reply]
Your script is different though, I'm not seeing `-e WEB_SERVER=false \`

**Reply by WoltervanDonk** [reply]
Sorry yeah in retried with the -e WEB_SERVER=false \ and it definitly did something. (i also figured out there was an issue with my token, which is now working correctly i think).

![image](https://user-images.githubusercontent.com/23165739/188219897-f54f80df-f3f4-47dd-b5fc-7c46beffc708.png)
The only thing it now says no files that can be imported, this is correct since i am using Nordigen. So how can i get to use Nordigen instead of the file location?

**Reply by JC5** [reply]
Even with Nordigen, it imports from that location. That location is where you need to put your configuration file (json).

**Reply by WoltervanDonk** [reply]
adding my nas path of the json to the PWD variable did the trick. Thank you so much, your a legend.

**Reply by JC5** [reply]
Thanks, I appreciate it!
